scripts/cfos/make_dataset_data_tables.py

```diff
@@ -143,57 +112,102 @@ def load_excel_input(excel, expected_dataset_cols):
         else:
             print(e)
         exit()
-    
-    print("Success: Excel file has been loaded into a dataframe.")
-    return dataset_metadata_df
 
+    print("Success: Excel file has been loaded into a dataframe.")
+    return raw_dataset_df
 
-def get_expected_columns_list(schema_json, dataset_name):
-    """Get list of expected columns in excel based on selected dataset schema."""
 
+def parse_config_file(schema_json, dataset_name):
+    """
+    Return the dictionary of schemas which define the tables and columns in the data, 
+    plus the dictionary of fields which contains all valid fields and their validation rules.
+    """
     # read schema into dictionary
-    with open(schema_json) as schema:
-        schema_dict = json.load(schema)
-
-    # get list of tables for a given dataset name
-    dataset_table_names = list(schema_dict[dataset_name].keys())
+    try: 
+        with open(schema_json) as schema:
+            config_dict = json.load(schema)
+    
+    except FileNotFoundError as e:
```

Comment:
I've been taught to do try/catch blocks and handle errors more completely than just letting it fail. I added these so I could go back and add more robust error handling, but it sounds like that isn't necessary for this. I'm still a bit hesitant about removing them without discussing it first because it still seems like good practice to have, but I'm not sure it's something we need here.

Reply:
Removed it